independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/handlers/FixedProducesHandler.java

```diff
@@ -40,8 +41,15 @@ public void handle(ResteasyReactiveRequestContext requestContext) throws Excepti
                 requestContext.setResponseContentType(mediaType);
                 requestContext.setEntityWriter(writer);
             } else {
-                throw new WebApplicationException(
-                        Response.notAcceptable(Variant.mediaTypes(mediaType.getMediaType()).build()).build());
+                // some clients might be sending the header with incorrect casing...
```

Comment:
[Technically](https://www.rfc-editor.org/rfc/rfc7231.html#section-3.1.1.1) not incorrect since it's supposed to be case-insensitive ;)

Reply:
Very good point :)